index.html

```diff
@@ -1657,36 +1657,24 @@ <h3>
             <dfn>auto</dfn>
           </dt>
           <dd>
-            <a data-lt=
-            "steps to programmatically determine the directionality of a member">
-            programmatically determined</a> from the value of the member.
+            no explicit directionality.
```

Comment:
```suggestion
            No explicit directionality.
```

Reply:
All of these start with a lowercase letter. For consistency, I made them all uppercase.